pkg/pillar/cmd/tpmmgr/tpmmgr.go

```diff
@@ -968,15 +1091,18 @@ func Run(ps *pubsub.PubSub) {
 		testTpmEcdhSupport()
 	case "testEcdhAES":
 		testEcdhAES()
-	case "createEkSrkAik":
+	case "createKeys":
 		if err = createKey(TpmEKHdl, tpm2.HandleEndorsement, defaultEkTemplate, false); err != nil {
-			log.Errorf("Error in creating Endorsement key: %v ", err)
+			fmt.Printf("Error in creating Endorsement key: %v ", err)
```

Comment:
Don't you want those logged so they have source etc set in Kibana? printf doesn't do that.

Reply:
This is for agent daemon debugging, i.e. to invoke tpmmgr as a command, so the intent is to print the output to stdout/stderr.

Reply:
OK